express-crud/server.ts

```diff
@@ -0,0 +1,116 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/grades', async (req, res, next) => {
+  try {
+    const sql = `
+      select *
+        from "grades";
+    `;
+    const result = await db.query(sql);
```

Comment:
We could also use generics here so we could annotate what the query is supposed to return. If you had this `type` at the top of the file

```
type Grade = {
  gradeId?: number,
  name: string,
  course: string,
  score: number, // A number between 0 and 100
};
```

Then you could use a generic like `const result = await db.query<Grade>(sql);`